Backpack-SwiftUI/Tests/Text/BPKTextTests.swift

```diff
@@ -40,10 +40,14 @@ class BPKTextTests: XCTestCase {
         assertSnapshot(BPKText("Hello Backpack, I hope you're having a great day!", style: .heading1))
     }
     
+    func test_withStyledText() {
+        assertSnapshot(BPKText("Hello **Backpack**, I hope you're having a _great_ day!", style: .heading1))
```

Comment:
Does this markdown formatting translate to iOS 14 as well?

Reply:
It does not, so It'd be up for the consumer in this case to determine if they should or should not use it.
Although we're getting rid of 14 now, so it does not worry me a whole lot! but up for debate!